supertokens/supertokens.go

```diff
@@ -119,44 +122,63 @@ func (s *SuperTokens) SendTelemetry() {
 
 func (s *SuperTokens) Middleware() http.HandlerFunc {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		reqURL, _ := NewNormalisedURLPath(r.URL.Path) // todo: error handle
+		reqURL, _ := NewNormalisedURLPath(r.RemoteAddr) // todo: error handle
 		path := s.AppInfo.APIGatewayPath.AppendPath(*reqURL)
-		// method := normaliseHttpMethod(r.Method)
+		method := r.Method
 
-		if strings.HasPrefix(path.GetAsStringDangerous(), s.AppInfo.APIBasePath.GetAsStringDangerous()) == false {
+		if !strings.HasPrefix(path.GetAsStringDangerous(), s.AppInfo.APIBasePath.GetAsStringDangerous()) {
 			return
```

Comment:
Are you sure `return` is the right thing to use here? In some web frameworks, don't you need to call the "next" middleware  API?